src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/ColorPickerTests/WinUIColorPickerPage.xaml.cs

```diff
@@ -0,0 +1,429 @@
+        {
+            if (spectrumRectangle != null && spectrumRectangle.Fill != null)
+            {
+                this.ColorSpectrumLoadedCheckBox.IsChecked = true;
+            }
+        }
+
+        private void PreviousColorRectangleFillChanged(DependencyObject o, DependencyProperty p)
+        {
+            if (previousColorRectangle != null)
+            {
+                SolidColorBrush previousColorBrush = previousColorRectangle.Fill as SolidColorBrush;
+
+                if (previousColorBrush == null)
+                {
+                    this.PreviousRedTextBlock.Text = "";
+                    this.PreviousGreenTextBlock.Text = "";
+                    this.PreviousBlueTextBlock.Text = "";
+                    this.PreviousAlphaTextBlock.Text = "";
+                }
+                else
+                {
+                    Color previousColor = previousColorBrush.Color;
+
+                    this.PreviousRedTextBlock.Text = previousColor.R.ToString();
+                    this.PreviousGreenTextBlock.Text = previousColor.G.ToString();
+                    this.PreviousBlueTextBlock.Text = previousColor.B.ToString();
+                    this.PreviousAlphaTextBlock.Text = previousColor.A.ToString();
+                }
+            }
+        }
+
+        private void SelectionEllipseStrokeChanged(DependencyObject o, DependencyProperty p)
+        {
+            UpdateSelectionEllipseColor();
+        }
+
+        private void UpdateSelectionEllipseColor()
+        {
+            if (selectionEllipse != null)
+            {
+                SolidColorBrush selectionEllipseStrokeBrush = selectionEllipse.Stroke as SolidColorBrush;
+
+                if (selectionEllipseStrokeBrush != null)
+                {
+                    Color selectionEllipseColor = selectionEllipseStrokeBrush.Color;
+
+                    this.EllipseRedTextBlock.Text = selectionEllipseColor.R.ToString();
+                    this.EllipseGreenTextBlock.Text = selectionEllipseColor.G.ToString();
+                    this.EllipseBlueTextBlock.Text = selectionEllipseColor.B.ToString();
+                    this.EllipseAlphaTextBlock.Text = selectionEllipseColor.A.ToString();
+                }
+            }
+        }
+
+        private void SelectionEllipsePositionChanged(DependencyObject o, DependencyProperty p)
+        {
+            UpdateSelectionEllipsePosition();
+        }
+
+        private void UpdateSelectionEllipsePosition()
+        {
+            if (selectionEllipse != null)
+            {
+                double ellipseX = Canvas.GetLeft(selectionEllipsePanel) + selectionEllipsePanel.Width / 2;
+                double ellipseY = Canvas.GetTop(selectionEllipsePanel) + selectionEllipsePanel.Height / 2;
+
+                this.EllipseXTextBlock.Text = Math.Round(ellipseX).ToString();
+                this.EllipseYTextBlock.Text = Math.Round(ellipseY).ToString();
```

Comment:
Codacy found an issue: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6121259)